app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/snapshot.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Snapshot < ::Snapshot
```

Comment:
Snapshots don't currently have a `"type"` column which means they don't support STI and subclassing.

Reply:
Sorry, but I need a little translation here. Without really knowing what this does and how it helps, I got this idea from: https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/app/models/manageiq/providers/vmware/cloud_manager/snapshot.rb
Do I not need this file at all?

Reply:
Oh sorry, if a model doesn't use STI then you can't subclass it so you can delete this file.  That VMware cloud manager one looks like a bug.